### PR TITLE
Default Linux VS Online Dev Container

### PR DESCRIPTION
**Pull request by Chuxel**
Implements #155. 

Currently uses `default` as the image name and sets the version to `0.1.0`.

### CODE REVIEW

**Comment by Chuxel** [comment]
I should also add that right now, this just installs Live Share and IntelliCode.  Do we also want to install all the language extensions supported?

File: containers/vsonline-linux/test-project/test.sh [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    checkMultiple "$1" 1 "[ -d ""$HOME/.vscode-server/extensions/$1*"" ]" "[ -d ""$HOME/.vscode-server-insiders/extensions/$1*"" ]" "[ -d ""$HOME/.vscode-test-server/extensions/$1*"" ]"
+}
+
+# -- Actual tests - add more here --
+checkMultiple "vscode-server" 1 "[ -d ""$HOME/.vscode-server/bin"" ]" "[ -d ""$HOME/.vscode-server-insiders/bin"" ]" "[ -d ""$HOME/.vscode-test-server/bin"" ]"
+check "non-root-user" "id vsonline"
+check "/home/vsonline" [ -d "/home/vsonline" ]
+check "sudo" sudo -u vscode echo "sudo works."
+check "git" git --version
+check "command-line-tools" which top ip lsb_release
```
Comment (derekbekoe) [inline containers/vsonline-linux/test-project/test.sh]: There are a few other checks we currently run. Would be good to add those as well.

```
# Check platforms
docker run $IMAGE oryx platforms
docker run $IMAGE python --version
docker run $IMAGE dotnet --info
docker run $IMAGE node --version
docker run $IMAGE php --version

# Check expected shells
docker run $IMAGE bash --version
docker run $IMAGE fish --version
docker run $IMAGE zsh --version
```

File: containers/vsonline-linux/.devcontainer/devcontainer.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+	"remoteUser": "vsonline",
+	"workspaceMount": "source=${localWorkspaceFolder},target=/home/vsonline/workspace,type=bind,consistency=cached",
+	"workspaceFolder": "/home/vsonline/workspace",
+	"runArgs": [ "--cap-add=SYS_PTRACE", "--security-opt", "seccomp=unconfined" ],
+
+	// Add the IDs of extensions you want installed when the container is created.
+	"extensions": [
+		"MS-vsliveshare.vsliveshare",
+		"VisualStudioExptTeam.vscodeintellicode"
+	],
```
Comment (derekbekoe) [inline containers/vsonline-linux/.devcontainer/devcontainer.json]: There are a couple other extensions we currently install. Should we include those as well?
```
"ms-vsonline.vsonline",
"GitHub.vscode-pull-request-github"
```

File: containers/vsonline-linux/test-project/test.sh (hunk 1 above)
Comment (Chuxel) [reply containers/vsonline-linux/test-project/test.sh]: Good idea - added!

File: containers/vsonline-linux/.devcontainer/devcontainer.json (hunk 2 above)
Comment (Chuxel) [reply containers/vsonline-linux/.devcontainer/devcontainer.json]: Done

**Comment by Chuxel** [comment]
@chrmarti and I chatted, and I think we've come around to JC's proposal of using "universal" as the image name rather than default.  We'll still have VSO in the description, which is the important part.

**Pull request merged**